app/styles/app.scss

```diff
@@ -3,3 +3,27 @@ $color: red;
 h1 {
   color: $color;
 }
+
+// stylelint-disable-next-line selector-pseudo-class-no-unknown
+:global(.home) {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 40px;
+  text-align: center;
+  line-height: 1.4;
+
+  a {
+    display: block;
+    margin: 0 0 20px;
+  }
+
+  p {
+    margin: 0 0 20px;
+  }
+
+  p:last-child {
+    margin-bottom: 0;
+  }
```

Comment:
Shouldn't we remove this line since there is only one `<p>` tag in the home template ?

Reply:
Yes, great point. That’ll simplify the boilerplate CSS as well!